migrations/20201004151420_case.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable func-names */
+/* eslint-disable no-sequences */
+const { tableCase } = require('../src/models/case.js')
+
+exports.up = function (knex) {
+  return knex.schema.createTable(tableCase, (table) => {
+    table.increments('CaseId').primary()
+    // eslint-disable-next-line no-unused-expressions
+    table.enum('RequestType', [
+      'Cooked Food',
+      'Diapers',
+      'Financial Assistance',
+      'Medical Bill',
+      'Milk Formula',
+      'School Fees',
+      'Transportation Fees',
+      'Utility Bill',
+    ]),
+      table.enum('Fulfilment', [
+        'In-Kind Donation',
+        'Cash Transfer',
+        'Third-Party Payment',
+        'Partner Referral',
+      ]),
+      table.text('Description'),
+      table.enum('CaseStatus', [
+        'New',
+        'On Hold',
+        'Referred to EFC',
+        'Processing',
+        'Closed',
+      ]),
+      table.enum('ReferenceStatus', ['Unverified', 'Pending', 'Verified']),
+      table.enum('Approval', ['-', 'Full', 'Partial', 'Rejected']),
+      table.decimal('AmountGranted'),
+      table.integer('BeneficiaryId').unique(),
+      table.integer('RefereeId').unique(),
```

Comment:
https://stackoverflow.com/questions/28350849/knex-migration-creating-foreign-key/29897797

Does this help?